checker/node/links.py

```diff
@@ -251,7 +251,9 @@ def run(cls, node: stac.Node) -> Iterator[stac.Issue]:
 
     if node.type == CATALOG:
       if CHILD not in links_by_rel:
-        yield cls.new_issue(node, f'{CATALOG} must at least one {CHILD} link')
+        yield cls.new_issue(
+            node, f'{CATALOG} must at least one {CHILD} link',
```

Comment:
"must contain at least one"?

Reply:
Done in https://github.com/google/earthengine-catalog/pull/305/commits/0700ecbf4089dc124851c6552f3590161b2cc978